### DISCUSSION TITLE
How to create orders page like on filament demo website?

### DISCUSSION THREAD

**Discussion by hellohendri** [filamentphp/filament#1981] [Help] [answered]
The main thing is I need a tutorial on how to make a orders page like at the demo website in correct way

I try to make a order page that almost the same as in filament website but my problem is :

1. Order Number always refresh when I try to select product or update the quantity
2. My repeater value is not detected when I click create button

Here is my Order model :

![order-model](https://user-images.githubusercontent.com/92468957/159700250-e4faacdc-2430-4c9d-b48b-54bb41b146c1.png)

Here is my Form :

![orders-form](https://user-images.githubusercontent.com/92468957/159700392-accc807c-8270-438b-bcf1-baec286c8b4a.png)

Also my another question is, did my method on creating this order page is correct? I mean like the column on my table, the way I use repeater, or etc.

Anyway I'am a totally beginner on laravel so any help will be so much helpfull 🙏🏻 and thanks to Filament Dev's for creating this awesome project!

**Comment by hellohendri** [comment, marked as answer]
https://github.com/laravel-filament/demo

**Reply by SvetoslavStefanov** [reply]
Thank you!
